js/modules/a11y-i18n.src.js

```diff
@@ -297,24 +297,26 @@ H.Chart.prototype.langFormat = function (langKey, context, time) {
 H.setOptions({
     lang: {
 
+        /* eslint-disable max-len */
```

Comment:
We could do line breaks with \ or + for the actual code (not sure if JSDoc handles line breaks in links), but that is a mess and easily breaks if you add whitespace, plus adds overhead to the code, so I suggest we keep the exception since the `max-len` rule is not making the code more readable here. But that is up for discussion. To me it seemed easier to do the exception for the whole block in case we want to make changes to the texts later, but we still do it block by block and not for the whole option set.